deeplinkdispatch-base/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkResult.java

```diff
@@ -29,6 +32,11 @@ public String error() {
     return error;
   }
 
+  /** @return the name of the component which is the DeepLink's destination. */
```

Comment:
`name of the component` isn't very helpful. Is this the fully qualified class name? can you give an example?